### PR TITLE
Feat/all cities endpoint

### PR DESCRIPTION
**Pull request by axefx**
Implements the fastapi endpoint 'all_cities' that queries all city information from database.
Validates type with pydantic BaseModel

### CODE REVIEW

File: project/app/api/city.py [hunk 1]
```diff
@@ -55,3 +58,40 @@ async def city(city_id: int):
...
+    JSON string containing 
+    - **city**: [str] . . . standard format city name, state name
+    - **pop**: [int] . . . population estimate
+    - **age**: [float] . . . average age of residents
+    - **income_household**: [int] . . . median household income
+    - **income_individual**: [int] . . . median individual income
+    - **home**: [int] . . . median home/condo price
+    - **rent**: [int] . . . median rent price
+    - **COLI**: [float] . . . Cost of Living Index [ACCRA Cost of Living Index](https://en.wikipedia.org/wiki/ACCRA_Cost_of_Living_Index)
+
```
Comment (ThomasMcDaniel91) [inline project/app/api/city.py:76]: Great doc string here

File: project/app/api/city.py [hunk 2]
```diff
@@ -55,3 +58,40 @@ async def city(city_id: int):
...
+                    'median_home_cost',
+                    'median_rent',
+                    'Cost_of_Living_Index']
+    citydata = fetch_query_records(query)
+    results = []
+    for i in citydata:
+        # uses schema to validate data type
+        city_dict = schemas.City(**dict(zip(columns,i)))
+        results.append(city_dict)
+    return results
```
Comment (ThomasMcDaniel91) [inline project/app/api/city.py:97]: Clean code. Everything looks good here

**Review by ThomasMcDaniel91 — APPROVED** [review]

**Review by AuFeld — APPROVED** [review]
LGTM

**Comment by SEE-DS** [comment]
I'm confused; 
after all the discussion and ambiguity, 
and with DS labs admin guidance to not do so, 
why code a whole-database endpoint?
what's the rationalization here? @ThomasMcDaniel91 @AuFeld @axefx

**Pull request merged**